src/translations/client.csv

```diff
@@ -1290,7 +1290,7 @@ form.section.accountDetails,,Account details,Coordonnées du compte
 form.section.assignedRegistrationOffice,,Assign to a registration office,A quel bureau voulez-vous affecter un nouvel utilisateur ?
 form.section.assignedRegistrationOfficeGroupTitle,,Assigned registration office,Bureau d'enregistrement assigné
 form.section.bride.headOfBrideFamily,,Head of bride's family,Chef de la famille de la mariée
-form.section.bride.name,,Print and issue to bride,Imprimer et envoyer à la mariée
+form.section.bride.name,,Bride,Mariée
```

Comment:
This change is perfectly fine but then I noticed there is a place in core which uses this translation (which it shouldn't).
https://github.com/opencrvs/opencrvs-
core/blob/develop/packages/client/src/forms/certificate/fieldDefinitions/collectorSection.ts#L1001-L1002
https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/forms/certificate/fieldDefinitions/collectorSection.ts#L627-L628
So we need keep the older translation but give it a separate id and use it on these 2 places